examples/history-api-fallback/file

```diff
@@ -0,0 +1 @@
+file
```

Comment:
I have done it, but now this path is not illustrative, it will work even without fix because active dotRule